addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/PathTraversalScanRule.java

```diff
@@ -659,6 +647,40 @@ public int getWascId() {
         return 33;
     }
 
+    private AlertBuilder createUnmatchedAlert(String param, String attack, String evidence) {
+        return newAlert()
+                .setConfidence(Alert.CONFIDENCE_LOW)
+                .setParam(param)
+                .setAttack(attack)
+                .setEvidence(evidence)
+                .setOtherInfo(Constant.messages.getString("ascanrules.pathtraversal.other", 5))
```

Comment:
IMO this can be removed now that it's using alert refs.

Reply:
Good point.